src/import/import-compress.c

```diff
@@ -351,6 +452,61 @@ int import_compress(ImportCompress *c, const void *data, size_t size, void **buf
                 break;
 #endif
 
+#if HAVE_ZSTD
+        case IMPORT_COMPRESS_ZSTD: {
+                /*
+                 * In theory, this says whether this is likely to be the last input chunk. However,
+                 * we have no reliable way of determining EOFs because our compression is streaming.
+                 *
+                 * NOTE: if this changes, be sure to call ZSTD_compressStream2(..., ZSTD_e_end) repeatedly
+                 * until it starts returning 0, i.e. it is allowed to call ZSTD_compressStream2(ZSTD_e_continue)
+                 * after ZSTD_compressStream2(..., ZSTD_e_end), but only after the latter call has returned 0
+                 * at least once.
+                 * */
+                const ZSTD_EndDirective mode = ZSTD_e_continue;
+                /* zstd-recommended size of the output buffer */
+                const size_t initial = ZSTD_CStreamOutSize();
```

Comment:
doesn't matter much, but we typically don't do `const` on local non-pointer variables. The compiler can figure this out on its own anyway. And it doesn't encode a contract of any kind with other stuff, since it's local variables after all, invisible to anyone outside this function.

but doesn't matter